routes/cartRouter.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const router  = express.Router();
+
+const {loginRequired} = require("../utils/auth")
+const { cartController } = require('../controllers');
+
+router.post('/',loginRequired, cartController.additionCart);
+router.get('/',loginRequired, cartController.getByCart);
+router.patch('/change', loginRequired,cartController.cartQuantityChange);
+router.delete('/all',loginRequired, cartController.allDeleteCart);
```

Comment:
전부를 의미할 때 굳이 all이라는 표현은 안 써도 될 것 같습니다.
```suggestion
router.delete('',loginRequired, cartController.allDeleteCart);
```

Reply:
말씀 듣고 보니 그런거 같아 수정반영하였습니다.